### PR TITLE
proxy-fixture

### PR DESCRIPTION
**Pull request by sandrampeter**
## Background

[Asana](https://app.asana.com/0/1200681690870921/1201744486584884/f)

Locally tested with terraform plan

### CODE REVIEW

**Review by anniehedgpeth — CHANGES_REQUESTED** [review]
I think you might still be testing, but I thought I'd throw in these little things real quick.

File: fixtures/test_proxy/variables.tf [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+  default     = null
+  type        = string
+  description = "The identity of the Key Vault which contains secrets and certificates."
+}
+
+variable "mitmproxy_ca_certificate_secret" {
+  default     = null
+  description = <<-EOD
+  The identifier of a secret comprising a Base64 encoded PEM certificate file for the mitmproxy Certificate Authority.
+  For GCP, the Terraform provider calls this value the secret_id and the GCP UI calls it the name.
```
Comment (anniehedgpeth) [inline fixtures/test_proxy/variables.tf:55]: Can you update this for Azure?

File: fixtures/test_proxy/variables.tf [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+  EOD
+  type        = string
+}
+
+variable "mitmproxy_ca_private_key_secret" {
+  default     = null
+  description = <<-EOD
+  The identifier of a secret comprising a Base64 encoded PEM private key file for the mitmproxy Certificate Authority.
+  For GCP, the Terraform provider calls this value the secret_id and the GCP UI calls it the name.
```
Comment (anniehedgpeth) [inline fixtures/test_proxy/variables.tf:64]: Can you update this for Azure?

**Review by anniehedgpeth — CHANGES_REQUESTED** [review]
Looks good! Just need to update a couple of variable descriptions for Azure that were copied from the GCP module.

File: fixtures/test_proxy/variables.tf (hunk 2 above)
Comment (sandrampeter) [reply fixtures/test_proxy/variables.tf]: done

File: fixtures/test_proxy/variables.tf (hunk 1 above)
Comment (sandrampeter) [reply fixtures/test_proxy/variables.tf]: done

**Review by anniehedgpeth — APPROVED** [review]

**Pull request merged**